Makefile

```diff
@@ -0,0 +1,55 @@
+GOLANGCI_VERSION=v1.28.3
+
+.PHONY: setup
+setup: ## Setup any developer dependencies
+	@echo "==>"
+	@echo "==> Installing dependencies..."
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_VERSION)
```

Comment:
you shouldn't need to install go lint by way of using a "tools" file https://github.com/10gen/baas/blob/master/etc/tools/tools.go such that you get it in the mod

Reply:
That's pretty nice, thanks for the tip!

Reply:
np! This is how we use it too `go run github.com/golangci/golangci-lint/cmd/golangci-lint run -v ./...` which works based on it being present in the module path

Reply:
FYI @edaniels golangci-lint advices against installing from source, I'm talking with mihai about rolling back this change now